### PR TITLE
webServer apiCfgs + nodeGroup part1

### PR DESCRIPTION
**Pull request by dragos-rebegea**
* web server api toml config file
* web server groups
* * node group

### CODE REVIEW

File: api/groups/nodeGroup.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+// nodeFacadeHandler defines the methods to be implemented by a facade for node requests
+type nodeFacadeHandler interface {
+	GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error)
+	GetClientInfo(client string) (string, error)
+	IsInterfaceNil() bool
+}
+
+// QueryDebugRequest represents the structure on which user input for querying a debug info will validate against
+type QueryDebugRequest struct {
```
Comment (iulianpascalau) [inline api/groups/nodeGroup.go:30]: we do not this yet

File: cmd/bridge/config/api.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# API routes configuration
+[APIPackages]
+
+[APIPackages.node]
+    Routes = [
+        # /node/status will return all metrics stored inside a node
+        { Name = "/status", Open = true },
+        # /node/peerinfo will return the p2p peer info of the provided pid
+        { Name = "/peerinfo", Open = true }
+    ]
```
Comment (iulianpascalau) [inline cmd/bridge/config/api.toml:20]: add empty line

File: cmd/bridge/main.go [hunk 3]
```diff
@@ -21,7 +22,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/cmd/node/factory"
 	elrondCommon "github.com/ElrondNetwork/elrond-go/common"
 	"github.com/ElrondNetwork/elrond-go/common/logging"
-	"github.com/ElrondNetwork/elrond-go/config"
+	goConfig "github.com/ElrondNetwork/elrond-go/config"
```
Comment (iulianpascalau) [inline cmd/bridge/main.go:25]: elrondConfig *

File: facade/relayerFacade.go [hunk 4]
```diff
@@ -25,6 +27,14 @@ func (rf *relayerFacade) PprofEnabled() bool {
 	return rf.pprofEnabled
 }
 
+func (rf *relayerFacade) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
```
Comment (iulianpascalau) [inline facade/relayerFacade.go:30]: missing comments on exported items.
Please add TODOs to implement these functions later.

File: relay/relay.go [hunk 5]
```diff
@@ -29,8 +30,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go-crypto/signing/ed25519"
 	"github.com/ElrondNetwork/elrond-go-crypto/signing/ed25519/singlesig"
 	logger "github.com/ElrondNetwork/elrond-go-logger"
-	"github.com/ElrondNetwork/elrond-go/api/shared"
-	"github.com/ElrondNetwork/elrond-go/config"
+	goConfig "github.com/ElrondNetwork/elrond-go/config"
```
Comment (iulianpascalau) [inline relay/relay.go:33]: elrondConfig *

File: relay/relay_test.go [hunk 6]
```diff
@@ -319,10 +298,32 @@ func TestRelay_CreateAndStartBridge(t *testing.T) {
 	})
 }
 
-func createMockDurationsMapConfig() ConfigStateMachine {
-	return ConfigStateMachine{
+func createMockRelay() Relay {
```
Comment (iulianpascalau) [inline relay/relay_test.go:301]: :+1:

**Review by iulianpascalau — APPROVED** [review]

**Review by mihaieremia — APPROVED** [review]

**Pull request merged**